meta-genivi-dev/recipes-sota/rvi-sota-client-append/rvi-sota-client_git.bbappend

```diff
@@ -0,0 +1,23 @@
+
+PR="r5"
+FILESEXTRAPATHS_prepend := "${THISDIR}:"
```

Comment:
FILESEXTRAPATHS_prepend := "${THISDIR}:" -> FILESEXTRAPATHS_prepend := "${THISDIR}:files" and move your toml and service files to "/files" directory